README.md

```diff
@@ -1 +1,308 @@
 # backend-capstone-template
+# ChatGPT Devs Era
+
+
+ChatGPT Devs Era is a website, customers can browse through various cooks and their menus, discovering a wide range of homemade food options. They can easily place orders for the meals they desire and have them delivered or arrange for pick-up.
+For Cooks, the platform provides a simple way to create profiles and showcase their homemade dishes. They can manage their menu, track orders, and interact with their customers.
+
+## Table of Contents
+
+- [Purpose](#purpose)
+- [Features](#features)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Examples](#examples)
+- [Contributing](#contributing)
+- [License](#license)
+
+
+## Purpose
+
+1.	Connect customers with cooks who offer homemade food.
+2.	Support local cookers and help customers who are looking for homemade food
+
+
+
+## Features
+
+1.	Connects customers with local cooks offering homemade food.
+2.	Browse and explore a diverse range of homemade dishes.
+3.	User-friendly interface for placing and managing orders.
+4.	Profiles for cooks to showcase their culinary offerings.
+5.	Order tracking and communication between customers and cooks.
+
+## Installation
+
+To install ChatGPT Devs Era, you can use yarn:
+
+```bash
+yarn install chatgpt-devs-era
+```
+
+### To start the server
+
+run the command `yarn && yarn start`. This will install all the dependencies
+
+### To run the tests
+
+run the command `yarn test`. Make sure that the server is up and running as well as the database
+
+### To install packages
+
+when you run `yarn add PACKAGE` the package will be installed in docker as well automatically. However if you run into issues, you need to stop the server first with `yarn run stop` then you can run `yarn run build` to rebuild the docker image and start again.
+
+## Usage
+
+To interact with the backend server, you can make HTTP requests to the following endpoints. The base URL for all endpoints is http://localhost:3000/api.
+
+Endpoints
+1. Create Normal User (Signup)
+Description: Creates a new normal user account.
+Endpoint: /auth/signup
+Method: POST
+Request Body:
+```json
+{
+    "firstname": "",
+    "lastname": "",
+    "email": "",
+    "password": "",
+    "role": ""
+}
+```
+
+2. Login Normal User
```

Comment:
can we just say User instead of Normal User? it looks better

Reply:
I agree, just saying user is more convenient

Reply:
it is ok